repos/spack_repo/builtin/packages/madgraph5amc/package.py

```diff
@@ -26,6 +26,7 @@ class Madgraph5amc(MakefilePackage):
     timeout = {"timeout": 60}
 
     with default_args(fetch_options=timeout):
+        version("3.6.2", sha256="1dde78f865278d8b9a59ca608076f495ad82cebe75f10469faba97210ddcdb6f")
```

Comment:
I am unable to confirm the sha256 and get `no remote versions` error when I try.

There is a newer version shown on the web site and it's sha256 is "207f990f7fb09164a54c5481e7fc6a41f000265b8ab3fdd77a844f505b06b169".

Reply:
I realize the check passed initially, but you can see from https://launchpad.net/mg5amcnlo/3.0/3.6.x that it is no longer listed.